packages/cbioportal-clinical-timeline/src/TimelineTracks.tsx

```diff
@@ -86,6 +86,7 @@ export const TimelineTracks: React.FunctionComponent<ITimelineTracks> = observer
                     return (
                         <Portal container={document.body}>
                             <Popover
+                                id="TimelineTracksTooltipPopover"
```

Comment:
is this to fix a warning?  i worry this gets transferred to dom id and we want to keep IDs out of DOM in general

Reply:
@alisman Yes, this fixes a warning that was showing in the browser console from prop_types. Specifically, "Warning: Failed prop type: The prop id is required to make Popover accessible for users of assistive technologies such as screen readers"

Honestly, I'm not familiar with screen readers or the  implication if you don't have an id here.